modules/samples/firealarm/component/api/src/main/java/org/homeautomation/firealarm/api/util/Constants.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.homeautomation.firealarm.api.util;
+
+import org.wso2.carbon.user.core.Permission;
+
+/**
+ * This hold the constants related to the device type.
+ */
+public class Constants {
+
```

Comment:
A single blank line is optionally before the first member or after the last member of the class (neither encouraged nor discouraged).
https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace